modules/spring-config/src/main/java/org/mule/runtime/config/spring/dsl/model/ApplicationModel.java

```diff
@@ -503,6 +506,25 @@ private boolean isMuleConfigurationFile() {
     return muleComponentModels.get(0).getIdentifier().equals(MULE_IDENTIFIER);
   }
 
+  private void validateErrorMappings() {
+    executeOnEveryComponentTree(componentModel -> {
+      List<ComponentModel> errorMappings = componentModel.getInnerComponents().stream()
+          .filter(c -> c.getIdentifier().equals(ERROR_MAPPING_IDENTIFIER)).collect(toList());
+      if (!errorMappings.isEmpty()) {
```

Comment:
Also, I realised I could have a general test using the PetStoreConnector so I've included that as well